src/main/webapp/cluster-view/cluster-view.html

```diff
@@ -126,28 +142,46 @@ <h4 style="color:   #009A31;">Available</h4>
 
             <!-- RIGHT COLUMN ROW #2 START -->
             <div class="row" style="padding-left:15px !important; margin: 0px important!;">
-                <div ng-show="shared.currentCollection == 'nodes'">
+                <div ng-show="isNodesView()">
                     <div ng-show="!currentCluster.getNodes() || currentCluster.getNodes().length === 0">There are no nodes to show.</div>
-                    <div class="box col-md-2 nopadding" ng-repeat="node in currentCluster.getNodes()">
-                        <a class="text-center" ui-sref="nodeDetails({clusterName: currentCluster.name, nodeName: node.name})">
-              {{node.name}} <br><br>
-            </a>
+                    <div class="box col-md-2 nopadding"
+                         ng-repeat="node in currentCluster.getNodes() | customPropertyFilter:'name':searchQuery"
+                         ng-show="node.show">
+                        <a class="text-center"
+                           ui-sref="nodeDetails({clusterName: currentCluster.name, nodeName: node.name})">
+                            {{node.name}} <br>
+                            <h6 ng-show="zoomedIn">5 caches</h6>
```

Comment:
Not so important, but I would maybe wrap these two header in `<div>` or `<span>` and use ng-show only on it, it will make more sense to do it if we there will be more values later.